homeassistant/components/tessie/sensor.py

```diff
@@ -296,3 +437,68 @@ def native_value(self) -> StateType | datetime:
     def available(self) -> bool:
         """Return if sensor is available."""
         return super().available and self.entity_description.available_fn(self.get())
+
+
+class TessieEnergyLiveSensorEntity(TessieEnergyLiveEntity, SensorEntity):
+    """Base class for Tessie energy site sensor entity."""
+
+    entity_description: SensorEntityDescription
```

Comment:
This is the default right? Or do we overwrite it somewhere?

Reply:
These entities use the base description.